azurerm/internal/services/applicationinsights/resource_arm_application_insights.go

```diff
@@ -89,6 +89,19 @@ func resourceArmApplicationInsights() *schema.Resource {
 
 			"tags": tags.Schema(),
 
+			"daily_cap": {
+				Type:         schema.TypeFloat,
+				Optional:     true,
+				Default:      100,
+				ValidateFunc: validation.FloatBetween(0, 1000),
+			},
+
+			"stop_send_notification_when_hit_cap": {
+				Type:     schema.TypeBool,
+				Optional: true,
+				Default:  false,
```

Comment:
Could we make this as computed and get the default from the api?

Reply:
Removed default switched to computed.